package-lock.json

```diff
@@ -9311,6 +9311,7 @@
     "regl-error2d": {
       "version": "2.0.6",
       "resolved": "git://github.com/gl-vis/regl-error2d.git#0d27313d932fdb8ffaed17931b7258958bce0b88",
+      "integrity": "sha1-2ydl+hP2aW6JTHSM16fEWhq9ouw=",
```

Comment:
@RussGlover can you run `npm i` again? This line shouldn't be here!

Reply:
I tried again but that line keeps getting changed

Reply:
What version of `npm` are you using?

Reply:
Its happening with 6.8.0 and 6.9.0. I was able to update it under npm in wsl and the integrity diff went away.

Reply:
I couldn't reproduce this behavior with npm 6.8.0 but I'm running Linux and it seems like you are on Windows :smile: Thanks again @RussGlover for reporting this issue!

Reply:
Okay everything should be good. I wasn't sure how to revert the bad commits and squash them into one commit without force pushing.